### PR TITLE
Change Stale Thresholds

### PR DESCRIPTION
**Pull request by Pandapip1**
## Abstract

The stale bot thresholds are too high. This PR reduces them.

## Motivation

Reduces EIP Editor work.

## Specification

1. The stale issue threshold was reduced from 6 months to four weeks
2. The stale PR threshold was reduced from two months to two weeks
3. The close PR threshold was *increased* from one week to two weeks
4. The stale PR label was changed to a more descriptive one

## Rationale

1. Encourages people to discuss relevant issues, and reduces the time it takes for irrelevant issues to disappear. Issues may be re-opened at any time if they become relevant again.
2. Encourages authors to update their EIP with editor-provided feedback or remind editors to review. PRs may be re-opened at any time if the author re-gains interest.
3. Since item 2 reduced the threshold so much, the PR close threshold was increased to give authors enough time to ensure that their PRs will not be mistakenly closed.
4. If the editors are actually the ones that need to review, the author will quickly make that clear, which will remove the stale tag.

### CODE REVIEW

**Comment by Pandapip1** [comment]
Blocked by https://github.com/ethereum/EIPs/pull/5365

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -20,17 +20,17 @@ jobs:
...
         days-before-issue-close: 7
         exempt-issue-labels: discussions-to,long-term
         stale-issue-label: stale
         # PR config
-        stale-pr-message: There has been no activity on this pull request in two months. It will be closed in a week if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback or request a review in a comment.
+        stale-pr-message: There has been no activity on this pull request in a week. It will be closed in two weeks if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback or request a review in a comment.
         close-pr-message: This pull request was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback or request a review in a comment.
-        days-before-pr-stale: 60
-        days-before-pr-close: 7
+        days-before-pr-stale: 7
```
Comment (SamWilsn) [inline .github/workflows/stale.yml:32]: I think this is too aggressive. I only review PRs once a week, and I've spent my time the last couple weeks making `eipw`.

Comment (Pandapip1) [reply .github/workflows/stale.yml:32]: How about two weeks?

**Comment by MicahZoltu** [comment]
These feel way to aggressive to me.  We very often have authors who take a month or two to apply feedback.

I'm curious what problem you are trying to solve with these shorter timelines.  A large number of open pull requests isn't a problem in itself, but perhaps it causes some problem for editors that I'm unaware of?

**Comment by Pandapip1** [comment]
> Perhaps it causes some problem for editors that I'm unaware of?

It's just a lot easier for an author to reopen closed PRs than it is to scan through 70+ PRs to find ones that I haven't yet reviewed. If an author takes 2 months to apply a change, then does it really need to be in the PR list the entire time? Once the change is made, reverting the stale bot is as simple as clicking the "Re-Open" button.

**Comment by MicahZoltu** [comment]
> It's just a lot easier for an author to reopen closed PRs than it is to scan through 70+ PRs to find ones that I haven't yet reviewed.

If we continue to iterate on and improve the labeling system, can this problem be addressed that way (so you can filter by label)?

What if we swapped the durations around so it was 2 weeks to flag as stale, then 6 months after that to close?  That way editors can filter out stale issues/PRs aggressively.

> If an author takes 2 months to apply a change, then does it really need to be in the PR list the entire time? Once the change is made, reverting the stale bot is as simple as clicking the "Re-Open" button.

One can imagine a stale bot that closes PRs after 24 hours for the same reasons you have listed.  I feel like 24 hours is "obviously too short".  What is unclear to me is what is the threshold for too short.  There are authors who check their EIPs only on the weekends, so one could argue for one week.  But sometimes people get busy on their weekend so maybe it takes them a few weeks.  Other times authors iterate outside of the EIP process and then show back up after much discussion to make updates to their PRs, like will probably be the case with #5353.

While your argument can stand for any duration, we ideally want to pick a value that results in the least amount of thrash (closing and opening PRs), else we might as well just have the bot auto-close after 24 hours so our PR list is perpetually empty.

**Comment by Pandapip1** [comment]
> What if we swapped the durations around so it was 2 weeks to flag as stale, then 6 months after that to close? That way editors can filter out stale issues/PRs aggressively.

I would be okay with that. That does indeed seem like the best option.

**Comment by MicahZoltu** [comment]
I think I'm OK with the latest iteration you have here.  Would like to get some feedback from other editors before merging though.

**Comment by Pandapip1** [comment]
It's been two days. Merging.

**Comment by MicahZoltu** [comment]
> It's been two days. Taking no signs of 👎 as a proxy for approval. Merging.

Please wait for at least one approval before merging.  Again, the bot is broken at the moment but we shouldn't be abusing that.  Also, we generally give other editors many weeks to provide feedback since I think you and I are the only two editors who look at things daily.  Most editors checkin weekly or monthly.

**Comment by MicahZoltu** [comment]
I reverted this change since it was merged prematurely (see #5397).  I recommend re-opening this PR as I think it is a good idea.

**Pull request merged**